.gitignore

```diff
@@ -105,4 +105,5 @@ venv.bak/
 
 #private files
 private/
-data-collecting/pubnub.ini
\ No newline at end of file
+data-collecting/pubnub.ini
+data-collecting/send-pubnub.py
```

Comment:
was this one on purpose? I think it will remove the python script from the repo

Reply:
You're right! I thought it would be a good idea to let it in .gitignore as default, because it contains real sensitive data. One possibility would be to rename send-pubnub.py to send-pubnub-template.py. What do you think?

Reply:
the private data is in the pubnub.ini config and not in the code. The pubnub.ini is in the gitignore, and we got a template that is public. So the send-pubnub.py has only code and reads from a private config file, pls check that that is the same in your setup

Reply:
Hi Tereza, you're right. I missunderstood what was written. You suggest to add the log file to gitignore and not the send-pubnub.py file. I'll fix this part. :)